pipeline/article_processing/controller.py

```diff
@@ -0,0 +1,196 @@
+from __future__ import division
+import itertools
+import pandas as pd
+import string
+from nltk.corpus import stopwords
+from nltk.tokenize import RegexpTokenizer
+from nltk.stem import SnowballStemmer
+from nltk import pos_tag
+from datetime import datetime
+
+
+# Manual invalid title and link
+maintenance_title = ['Reduced service at Agrimoney.com',
```

Comment:
Ok, any idea when it will be finished? I will leave this here for now, once you have completed and tested the result, then we can remove this.